app/models/ems_refresh/save_inventory.rb

```diff
@@ -213,8 +214,9 @@ def save_disks_inventory(hardware, hashes)
 
     # Update the associated ids
     hashes.each do |h|
-      h[:storage_id] = h.fetch_path(:storage, :id)
-      h[:backing_id] = h.fetch_path(:backing, :id)
+      h[:storage_id]      = h.fetch_path(:storage, :id)
+      h[:backing_id]      = h.fetch_path(:backing, :id)
+      # h[:storage_profile] = h.fetch_path(:storage_profile, :id)
```

Comment:
Is this meant to be commented out?  If not can probably set the padding back on the previous two lines.